README.md

```diff
@@ -12,20 +12,24 @@ npm install feathers-mongodb-fuzzy-search
 ```js
 const search = require('feathers-mongodb-fuzzy-search')
 
-// get service
-const messages = app.service('messages')
-
-// enable text index on title property, makes the title property searchable
-messages.Model.createIndex({ title: 'text' })
-
-// add fuzzy search hook, may also use app.hooks for all services
-messages.hooks({
+// add fuzzy search hook, may also use service.hooks to apply it on individual services only
+app.hooks({
   before: {
     find: search()
   }
 })
 
-// find documents
+// Field matching
+const users = app.service('users')
+// find users with first name containing a 's' and last name starting by 'art'
+let userDocuments = await users.find({ query: { firstName: { $search: 's' }, lastName: { $search: '^art' } })
+
+// Full-text search
+const messages = app.service('messages')
+// enable text index on title property, makes the title content searchable
+messages.Model.createIndex({ title: 'text content talking about cats' })
```

Comment:
I'm pretty sure this is the wrong use of `createIndex`: https://docs.mongodb.com/manual/reference/method/db.collection.createIndex/

Reply:
Yes you're right my bad